server/src/main/auth/auth.routing.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express'
+import { register, saveAndSend } from './register/register'
+import { verify } from './verify/verify'
+import { signin } from './signin/signin'
```

Comment:
register、saveAndSend、verify、signin 這幾個可以不用個別是 controller ，因為他們邏輯很相關，可以是同個 controller 底下的不同個 action 。
```suggestion
import controller from './auth.controller'
// controller.register、controller.saveAndSend
// controller.verify
// controller.signin